demos/demo_tricamera.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+"""
+This demo is to start three pylon dependent cameras and store a list
+of 100 observations obtained from them as a pickle file.
+"""
+import numpy as np
+import cv2
+import pickle
+
+import robot_interfaces
+
+
+def main():
+
+    camera_data = robot_interfaces.tricamera.Data()
+    camera_driver = robot_interfaces.tricamera.SyncDriver("cam_1", "cam_2", "cam_3")
```

Comment:
For testing on the TriFinger platform: the cameras there are called "camera60", "camera180" and "camera300" (rougly refering to their angle like in the naming of the fingers)